package.json

```diff
@@ -20,7 +20,7 @@
 		"build": "rimraf lib && babel src --out-dir lib --copy-files",
 		"coveralls": "cat ./reports/lcov.info | coveralls",
 		"create": "node ./scripts/create-rule",
-		"flow": "if [ ! -e ./.flowconfig ]; then echo \"Could not find .flowconfig\"; else flow; test $? -eq 0 -o $? -eq 2; fi",
+		"flow": "flow",
```

Comment:
shouldn't this be `flow check`? 🤔

Reply:
`flow check` can only be used after `flow start` to spawn the server. this is a one-hit command that does both 😇

Reply:
TIL